airbyte-integrations/bases/base-java/build.gradle

```diff
@@ -7,6 +7,8 @@ dependencies {
     implementation 'commons-cli:commons-cli:1.4'
 
     implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-queue')
```

Comment:
Why did we add the deps to `airbyte-queue`?

Reply:
Hmmm...as part of this PR I explicitly added the only implementation project(":airbyte-json-validation").
Have no idea where "airbyte-queue" comes from. Maybe as part of auto-merging masker branch... some kind of merge conflicts but I didn't see it. Shall we remove it?

Reply:
I'm removing this here: https://github.com/airbytehq/airbyte/pull/4881/files#r673985881